packages/web-components/src/book-valuation-widget/client/core/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -16,9 +16,11 @@ Book_valuation_widget {
       [Function],
       [Function],
       Object {
+        "advanceSearchButton": "css-q9ua3x",
```

Comment:
Should be featureButton as per my comment

Reply:
updated